storm/src/main/storm/mesos/NimbusScheduler.java

```diff
@@ -102,7 +102,6 @@ public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
       case TASK_LOST:
       case TASK_ERROR:
         LOG.info("Received status update: {}", taskStatusToString(status));
```

Comment:
@srishtyagrawal 

> should I remove TASK_ERROR and TASK_STARTING?

Nope, perhaps I need to give more info:

1. These status updates can also come from the mesos-agent/slave & mesos-master (maybe?), not *just* from `MesosSupervisor`.  So you not seeing them being sent from the `MesosSupervisor` is not proof that they won't happen.
2. If you just removed those cases, then when we did receive those status updates we would fall through into the `Received unrecognized status` default case.